src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -236,16 +284,16 @@ public int getTotalPageCount() {
 	 * Gather additional info about the logged in user
 	 */
 	public void sendJoinRequestStep1(WikiPageKey challengeInfoWikiPageKey) {
-		UserSessionData sessionData = authenticationController.getCurrentUserSessionData();
-		UserProfile profile = sessionData.getProfile();
-		view.updateWizardProgress(currentPage, getTotalPageCount());
-		view.showChallengeInfoPage(profile, challengeInfoWikiPageKey, new Callback() {
+		progressWidget.configure(currentPage, getTotalPageCount());
```

Comment:
Verify Challenge Info page is shown (if isChallenge=true), and not shown if isChallenge=false.